### DISCUSSION TITLE
Question about task number in FIXED stages

### DISCUSSION THREAD

**Discussion by eedalong** [trinodb/trino#15321] [Q&A]
I see that FIXED stages [always schedule 100 tasks](https://github.com/prestodb/presto/blob/445d33f05b50a2daa6bf2e79bef1a4d6c80fab03/presto-native-execution/presto_cpp/presto_protocol/special/SystemPartitioningHandle.java#L149) for each one right? If we use all-at-once schedule policy and my cluster doesnt have enough resoures, this means I cant start my query right?

**Comment by hashhar** [comment]
Your link points to something outside of Trino so I'm not sure if the code you're looking at actually matches what Trino does.
